tests/device_tests/test_msg_getaddress.py

```diff
@@ -269,15 +269,14 @@ def test_unknown_path_tt(client):
 def test_unknown_path_t1(client):
```

Comment:
It should be possible to drop this testcase and use `test_unknown_path_tt` for both models except that `get_address` behaves differently with strict setting and `show_display=False`. In this case TT validates the path but T1 doesn't. Or we can cheat and use `show_display=True` in the call below `# account number is too high`.

Reply:
Same with `test_invalid_path`.

Reply:
Done in f97c6e2e9c6e4d7900dae78ca58236d0990482a7. I changed to `show_display=True` so that we can unify the tests. In terms of security `show_display=True` if the more important case anyway.